interp/type.go

```diff
@@ -1350,7 +1365,7 @@ func (t *itype) lookupMethod(name string) (*node, []int) {
 				}
 			}
 		}
-		if t.cat == aliasT {
+		if t.cat == aliasT || isInterfaceSrc(t) && t.val != nil {
```

Comment:
Did you notice that in `lookupFieldOrMethod` (which calls `lookupMethod`), we already deal with the case of `t.cat == interfaceT`? So I think there's probably some more refactoring we could related to these calls. But I understand if you want to do it later.

Reply:
Yes I noticed, and indeed intend a refactoring to be done after this PR